package.json

```diff
@@ -183,8 +183,8 @@
     "punycode": "^2.1.1",
     "puppeteer": "^9.1.1",
     "raf": "^3.4.1",
-    "react": "16.4.2",
-    "react-dom": "16.4.2",
+    "react": "16.8.3",
+    "react-dom": "16.8.3",
```

Comment:
I made such an attempt, but it will cause the map component to fail to run normally and report an error because the main version of react in the project v16.3 does not support hooks😞. So I update it.